CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ created.
 ### Changed
 - ([#112](https://github.com/microsoft/hi-ml/pull/112)) Update himl_tensorboard to work with files not in 'logs' directory
 - ([#106](https://github.com/microsoft/hi-ml/pull/106)) Split into two packages. Most of existing package renamed to hi-ml-azure, remained remains hi-ml.
+- ([#113](https://github.com/microsoft/hi-ml/pull/113) Add helper function to download files from AML Run)
```

Comment:
please add to changelog that some functions were moved, and commandline args to script?